src/meta/src/hummock/manager/compaction/compaction_group_schedule.rs

```diff
@@ -1006,32 +989,34 @@ pub fn is_table_low_write_throughput(
 }
 
 fn check_is_low_write_throughput_compaction_group(
-    table_write_throughput: &HashMap<u32, VecDeque<u64>>,
-    sample_size: usize,
-    threshold: u64,
+    table_write_throughput_statistic_manager: &TableWriteThroughputStatisticManager,
     group: &CompactionGroupStatistic,
-    low_write_throughput_ratio: f64,
+    opts: &Arc<MetaOpts>,
 ) -> bool {
-    // check table exists
-    let live_table = group
+    let table_with_statistic: Vec<_> = group
         .table_statistic
         .keys()
-        .filter(|table_id| table_write_throughput.contains_key(table_id))
-        .filter(|table_id| table_write_throughput.get(table_id).unwrap().len() >= sample_size)
+        .filter(|table_id| table_write_throughput_statistic_manager.contains(**table_id))
```

Comment:
I think this check is no longer needed because non-existing tables can be filtered inside `get_table_throughput`.